pkgs/development/tools/misc/coccinelle/default.nix

```diff
@@ -1,21 +1,40 @@
-{ fetchurl, lib, stdenv, python3, ncurses, ocamlPackages, pkg-config }:
+{ fetchFromGitHub, fetchpatch, lib, stdenv, python3, ncurses, ocamlPackages, pkg-config, autoreconfHook }:
 
 stdenv.mkDerivation rec {
   pname = "coccinelle";
-  version = "1.1.0";
+  version = "1.1.1";
 
-  src = fetchurl {
-    url = "https://coccinelle.gitlabpages.inria.fr/website/distrib/${pname}-${version}.tar.gz";
-    sha256 = "0k0x4qnxzj8fymkp6y9irggcah070hj7hxq8l6ddj8ccpmjbhnsb";
+  # The release tarball is unable to fetch so use github mirror insead (see NixOS/nixpkgs #171019)
+  src = fetchFromGitHub {
+    repo = pname;
+    rev = version;
+    owner = "coccinelle";
+    hash = "sha256-rS9Ktp/YcXF0xUtT4XZtH5F9huvde0vRztY7vGtyuqY=";
   };
 
-  buildInputs = with ocamlPackages; [
-    ocaml findlib menhir
-    ocaml_pcre parmap stdcompat
-    python3 ncurses pkg-config
+  patches = [
+    # Fix data path lookup.
+    # https://github.com/coccinelle/coccinelle/pull/270
+    (fetchpatch {
+      url = "https://github.com/coccinelle/coccinelle/commit/2fb5fd176c3a8b79b6b3648bdffe9c6761f3136c.patch";
```

Comment:
We might want to use https://github.com/coccinelle/coccinelle/commit/540888ff426e0b1f7907b63ce26e712d1fc172cc instead.